homebytwo/importers/models/switzerlandmobilityroute.py

```diff
@@ -132,7 +131,7 @@ def get_route_data(self, cookies=None, raw_route_json=None):
             )
 
             # create geom from lat, lng data columns
-            coords = [(lat, lng) for lat, lng in zip(data["lat"], data["lng"])]
+            coords = zip(data["lat"], data["lng"])
```

Comment:
Nice catch, I saw it last week and did not fix it. 😘 
There is even another useless list comprehension somewhere else in the code if I remember well.

Can you spot it?

Reply:
But it does seem to break the tests...

Reply:
Ah probably because it returns an iterator. Try surrounding it with `list()`.

Reply:
Works indeed. Spotted the other one as well: https://github.com/HomebyTwo/homebytwo/pull/73/commits/c72720afb05244f487d4e487ebba056405a92e50#diff-fecdbf464accb987a890345b98a2724dR118